h2o-core/src/main/java/water/fvec/ChunkUtils.java

```diff
@@ -0,0 +1,23 @@
+package water.fvec;
+
+
+/**
+ * Simple helper class which publishes some package private methods as public
+ */
+public class ChunkUtils {
```

Comment:
Yup.

There are more options - either make these methods public instead of package private and remove the ChunkUtils here and FrameUtils in Sparkling-Water.

Otherwise I would remove `ChunkUtils` to `FrameUtils` in h2o, remove `FrameUtils` from Sparkling Water and move the remaining methods to ChunkUtils:
 - `preparePartialFrame`
- `finalizePartialFrame`
- `getChunksLocations` with a small change that this method would return Array[H2ONode] instead of Array[NodeDesc]

So this way we would have single place where to have these methods. What do you think ?

If the first option ( making the methods public ) is not an option, then I'm up to completely move the rest of FrameUtils on the H2O side.